en/developer-docs/docs/manage-databases-and-caches/choreo-managed-databases-and-caches.md

```diff
@@ -3,13 +3,42 @@
 Choreo allows you to create PostgreSQL and MySQL databases as well as Choreo-Managed Cache instances on all major cloud providers (AWS, Azure, GCP, and DigitalOcean) as fully Choreo-managed platform services.
 These databases and caches can be seamlessly provisioned to offer persistence and caching capabilities for all your Choreo components. Choreo provides various service plans for each type, ranging from smaller instances for development purposes to production-grade databases with automatic backups and high-availability multi-nodes.
 
+## Technology Partnership
+
+Choreo's managed data services are powered by Aiven, our trusted technology partner and data platform subprocessor. Aiven provides the underlying infrastructure, automated management, and expertise that enables Choreo to offer these fully managed data services within the Choreo ecosystem
```

Comment:
Do we need to add a link here?

Reply:
<img width="590" height="478" alt="image" src="https://github.com/user-attachments/assets/7d77e14f-7633-4b84-be56-bb8d4a1ec0a7" />

I don't think we need to link aiven.io in all these places - but if you think it's necessary happy to add them.

Reply:
Not in all the places, but when we introduce Aiven as our technology partner I think it's better if we add a link